src/views/ManageList.jsx

```diff
@@ -1,7 +1,108 @@
+import { useCallback, useState } from 'react';
+import { useStateWithStorage } from '../utils';
+import { addItem } from '../api';
+
+const soonDate = 7;
+const kindOfSoonDate = 14;
+const notSoonDate = 30;
```

Comment:
Nit: Typically `date` in a name suggests it's some sort of `Date` object, or like an ISO date. Since these are just number of days, I'd opt to use "days" as the descriptor here.

```suggestion
const soonDays = 7;
const kindOfSoonDays = 14;
const notSoonDays = 30;
```